src/lib/viewers/controls/icons/IconDrawing24.tsx

```diff
@@ -1,9 +1,13 @@
 import * as React from 'react';
 
 function IconDrawing24(props: React.SVGProps<SVGSVGElement>): JSX.Element {
+    const { fill } = props;
```

Comment:
Can this be inlined with the function declaration? Do we want to spread `fill` onto both elements or should we use `...rest` for the SVG?

Reply:
Yes to inlining, yes for using ...rest as well.